tests/unit/modules/test_pdbedit.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+# Import python libs
+from __future__ import absolute_import, print_function, unicode_literals
+
+# Import Salt Libs
+import salt.modules.pdbedit as pdbedit
+
+# Import Salt Testing Libs
+from tests.support.mixins import LoaderModuleMockMixin
+from tests.support.unit import TestCase
+
+
+class PdbeditTestCase(TestCase, LoaderModuleMockMixin):
+    '''
+    TestCase for salt.modules.pdbedit module
+    '''
+
+    def setup_loader_modules(self):
+        return {pdbedit: {}}
+
+    def test_generate_nt_hash(self):
```

Comment:
For #55185 I will need a similar test but need to mock the result of cmd.run... can I still include the mock data in the test file or does it need to go into tests.support.<something> ?

Reply:
the test file should work